source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,27 @@ The Syscollector module supports different options across various operating syst
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
 |    Solaris             |     ✗     |     √     |     √     |     √    |     ✗     |     ✗     |     –     |
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
+
+.. versionadded:: 4.7.0
+
+The Syscollector module now supports scan packages from `NPM (Node Package Manager)  <https://https://www.npmjs.com/>`_, `PyPI (Python Package Index)  <https://https://www.npmjs.com/>`_ and `Snap store <https://snapcraft.io/>`_ , **the support is only for the default paths of those packages' installation**.
+
+The following table shows the new feature that are compatible with various operating systems, where '*√*' means compatible enhancement, '*✗*' not supported enhancement, and '*–*' not applicable.
+
++------------------------+--------------------------------+
+|                        |  **Syscollector enhancement**  |
++  **Operating System**  +------+-------------+-----------+
+|                        |  NPM |    PyPI     |  Snap     |
```

Comment:
I wouldn't mention Snap, since it is not part of 4.7.0

Reply:
Talking about Snap seems out of scope.